### ISSUE TITLE
Angle brackets in object key names aren't escaped properly

### ISSUE THREAD

**Issue by bovine3dom** [TypeStrong/typedoc#780]
```ts
/** important object, document me */
const foo = {
    "<c-a>": "yes, I really used that as a key name",
}
```
gets turned into
```html
[...]
<div class="tsd-signature tsd-kind-icon"><c-<wbr>6&gt;<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;"yes, I really used that as a key name"</span></c-<wbr></div>
[...]
```

which renders as

```
a>: = "yes, I really used that as a key name"
```

rather than the desired

```
<c-a>: = "yes, I really used that as a key name"
```

I realise it's probably pretty niche, so I'd be happy to have a crack at fixing it myself if anyone could point me in the right direction.

Thanks!

**Comment by Gerrit0** [comment]
Leaving this open to track release, fixed in https://github.com/TypeStrong/typedoc/commit/5355abc164d5db8924698e2aef1190a081d0b659

**Comment by Gerrit0** [comment]
Fixed in 0.15.1 :)
